### DISCUSSION TITLE
Love all the help features

### DISCUSSION THREAD

**Discussion by JonathanAquino** [jesseduffield/lazygit#1174] [General]
Just want to say I'm really impressed by and grateful for all the help features that lazygit has. The "x" help and the shortcut list at the bottom of the screen are so helpful. Thanks for this.

**Comment by jesseduffield** [comment]
@JonathanAquino no worries! I plan on adding some more help feature very soon: including a commands log https://github.com/jesseduffield/lazygit/pull/1249

and a 'tip of the day' thing which will sit at the top of the that log
